### ISSUE TITLE
Kopnięcie po okręgu - idea

### ISSUE THREAD

**Issue by silent-rock** [silent-rock/nwn_BS#42]
Atut w wersji vanilla zupełnie bezużyteczny + zawiera parę bugoli.

Ciekawe rozwiązanie: https://neverwintervault.org/project/nwn1/module/circle-kick-mode

**Comment by blashkow** [comment]
na wm bylo to poprawione?  nie zauwazylem tego żeby coś tam sie dzialo.

**Comment by silent-rock** [comment]
Nie, nie spotkałem się z serwerem, na którym ten atut by był poprawiony.
